otx/algorithms/common/adapters/mmcv/configs/backbones/deeplabv3plus_resnet50.py

```diff
@@ -0,0 +1,47 @@
+norm_cfg = dict(type="BN", requires_grad=True)
```

Comment:
I hope you align the name "resnet50_deeplabv3plus" like template name.

Reply:
I don't know what is right, but maybe deeplabv3plus_resnet is more appropriate because names of old templates are ocr_lite_hrnet_XX(backbone name later). Please align them.

Reply:
OK